.github/workflows/CI.yml

```diff
@@ -9,19 +9,19 @@ concurrency:
   # Skip intermediate builds: always.
   # Cancel intermediate builds: only if it is a pull request build.
   group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+  cancel-in-progress: false
```

Comment:
Canceling runs in progress sounds useful, but in practice usually the following. One run finishes after running all the tests and detected a failing test, so the job reports a failure. All other jobs are at around 90% and get cancelled resulting in zero useful information. So, basically, the 90% of time spent on running tests is wasted when cancel-in-progress is `true`.